bayes/src/main/java/com/enigmastation/ml/bayes/annotations/BayesClassifier.java

```diff
@@ -19,6 +19,9 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 
+/**
```

Comment:
These are identifiers for use in DI frameworks. Good point. Need to demonstrate usage.